exercises.ipynb

```diff
@@ -21,29 +20,536 @@
     "import requests"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 2"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 257,
-   "metadata": {
-    "autoscroll": false,
-    "collapsed": false,
-    "ein.hycell": false,
-    "ein.tags": "worksheet-0",
-    "slideshow": {
-     "slide_type": "-"
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "START_URL = 'https://brickset.com/sets/year-2016'\n",
+    "result = requests.get(START_URL)\n",
+    "soup = BeautifulSoup(result.content, \"html.parser\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_links(soup) : \n",
+    "    nexturl =[]\n",
```

Comment:
Proper names! This is not singular, there are many urls! So it should be next_urls!